extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestWorkerClient.java

```diff
@@ -138,12 +133,11 @@ public ListenableFuture<Boolean> fetchChannelData(
     catch (Exception e) {
       throw new ISE(e, "Error reading frame file channel");
     }
-
   }
 
   @Override
   public void close()
   {
-    inMemoryWorkers.forEach((k, v) -> v.stopGracefully());
+    inMemoryWorkers.forEach((k, v) -> v.stop());
```

Comment:
Anything which prompted this change?

Reply:
I thought that `stopGracefully` was not really an appropriate method name to have on `Worker`. It's too linked to the behavior of tasks, and I wanted to make the `Worker` interface more easily extensible to workers that aren't tasks.